blscurve/bls.nim

```diff
@@ -32,6 +33,12 @@ const
   RawVerificationKeySize* = MODBYTES_384
   RawSignatureSize* = MODBYTES_384 * 2
 
+proc init*(T: SigKey | VerKey): auto =
+  ## Initialize ``SignatureKey``, ``VerificaitonKey`` to the infinitiy point
+  var ret: T
+  ret.point.inf()
+  ret
```

Comment:
When we do `echo SigKey()` we do get the infinity point though. Is there a disconnect between `getBytes`/hex serialization and the actual memory representation?

Reply:
`inf` does stuff: https://github.com/status-im/nim-blscurve/blob/master/blscurve/csources/64/ecp_BLS381.c#L149

I'm not sure how the serialization works exactly, but it seems prudent to make this call here (rust version does a similar thing)

Reply:
I think when we do SigKey() we get a full zero uncompressed signature, but then when we compress it, we add the compressed bit, which gets serialized as 0xc0000... into hex